data/newMigrations/003-service_categories.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Everything looks great, except somehow this line has snuck back-in in both files now.

Reply:
I put those in to take screen shots -- the eslint "prettier" puts errors on every line, I think because I use four-space indents and it wants two -- and didn't realize it was still there on the push.  Sorry!  Removed now.